backend/src/test/java/com/funeat/acceptance/recipe/RecipeAcceptanceTest.java

```diff
@@ -39,11 +44,46 @@ public class RecipeAcceptanceTest extends AcceptanceTest {
         STATUS_CODE를_검증한다(response, 정상_생성);
     }
 
+    @Test
+    void 레시피의_상세_정보를_조회한다() {
+        // given
+        카테고리_추가_요청(간편식사);
```

Comment:
요거는 충돌 해결할 때 편하게 수정하기 위해서 아래처럼 바꿔줄 수 있나여?

```suggestion
        final var category = new Category("간편식사", CategoryType.FOOD);
        카테고리_추가_요청(category);
```